### ISSUE TITLE
Shadow under the video.

### ISSUE THREAD

**Issue by dma3000** [olive-editor/olive#595]
I think it would be useful to make a shadow effect under the video.

![olive_5](https://user-images.githubusercontent.com/36135809/53755296-3e879e00-3ec7-11e9-971c-227a0eecc948.png)

**Comment by DaniSeeh** [comment]
Could you be a bit more specific? Like, in the UI? Or add an effect that would generate a drop shadow? You could already do this pretty easily by duplicating the clip, scaling it over, and changing the color to black and making it transparent. I could show you how if you need a quick tutorial

**Comment by dma3000** [comment]
> You could already do this pretty easily by duplicating the clip, scaling it over, and changing the color to black and making it transparent.

Yes, you can do it, but if the clip moves, then the shadow needs to be moved.
I think it is better to make exactly the effect that allows you to create a shadow for the clip.

**Comment by itsmattkc** [comment]
There is actually a work-in-progress effect of this nature, but it does require an extra hook in the effect processing code so that the whole video gets shadowed rather than just the contents inside it. But yeah, it's coming, no ETA yet though.

**Comment by Symbian9** [comment]
TL;DR: There is an easy solution by just create such effect using "image with shadow" and combine sequences by including one to another.

I would publish example later today.

**Comment by Symbian9** [comment]
Here is example on adding shadow using external image file

![pic.1](https://i.imgur.com/BbcUJrh.png)

Unpack both files from ZIP into same folder and open *Olive*

- https://0x0.st/z8HT.zip

P.S.: Image used as shadow could be created using any image editor that has blur effect and support transparency; I create it using *AzPainter*

- https://github.com/Symbian9/azpainter

**Comment by itsmattkc** [comment]
This will be addressed in the node rewrite.
